### PR TITLE
384 Ciphersuite Addition

### LINKED ISSUE #503: Ciphersuite Addition
I suggest that we consider including a ciphersuite that enables AES256 GCM with SHA384 and that for signatures ECDSA with P-384 (and SHA-384) is also permitted. The NSA restricts commercial applications to certain algorithms for applications that are used in or in coordination with the U.S. federal government and, from a general availability perspective, adding this could expand protocol use potential.

### PR DESCRIPTION
**Pull request by br-hale**
Fixes #503

### CODE REVIEW

File: draft-ietf-mls-protocol.md [hunk 1]
```diff
@@ -3627,6 +3627,7 @@ Initial contents:
 | 0x0004          | MLS10_256_DHKEMX448_AES256GCM_SHA512_Ed448            | Y           | RFC XXXX  |
 | 0x0005          | MLS10_256_DHKEMP521_AES256GCM_SHA512_P521             | Y           | RFC XXXX  |
 | 0x0006          | MLS10_256_DHKEMX448_CHACHA20POLY1305_SHA512_Ed448     | Y           | RFC XXXX  |
+| 0x0007          | MLS10_256_DHKEMP384_AES256GCM_SHA384_P384.           | Y           | RFC XXXX  |
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md:3630]:
```suggestion
| 0x0007          | MLS10_256_DHKEMP384_AES256GCM_SHA384_P384.            | Y           | RFC XXXX  |
```

**Review by bifurcation — CHANGES_REQUESTED** [review]
@br-hale I may have been over-optimistic when I said this was a one-line PR :)  Could you please update the tables below the change you made? You need a row of the following form:

```
| Value  | KEM    | KDF    | AEAD   | Hash   | Signature              |
|:-------|:-------|:-------|:-------|:-------|:-----------------------|
| 0x0007 | 0x0011 | 0x0002 | 0x0002 | SHA384 | ecdsa_secp384r1_sha384 |
```

... and to update this text:

>  In the ciphersuites defined above, "SHA256" and "SHA512" refer to the SHA-256 and SHA-512 functions defined in {{SHS}}.

With those two changes, I think this is good to merge.

File: draft-ietf-mls-protocol.md (hunk 1 above)
Comment (br-hale) [reply draft-ietf-mls-protocol.md:3630]: You committed to a one-line change! 
No problem - changes made, as well as a typo correction that was lingering in the spec.

**Review by bifurcation — APPROVED** [review]

**Review by raphaelrobert — APPROVED** [review]
If this is needed for the reasons mentioned in the issue, I have no objections.

**Pull request merged**